### DISCUSSION TITLE
'silent' write transactions

### DISCUSSION THREAD

**Discussion by joranmulderij** [isar/isar#401] [Ideas]
Could you please update the documentation to explain what the 'silent' argument on the write transaction method does? Other parts of the documentation are also either missing or inaccurate. Please make sure the documentation is at least accurate.

**Comment by simc** [comment]
`silent = false` does not trigger object/collection/query watchers.

> Other parts of the documentation are also either missing or inaccurate

What are you missing? Feel free to create an issue or PR to fix the docs.

>  Please make sure the documentation is at least accurate.

I don't like the sound of this. Fiy: I work on this project in my free time and without compensation.

**Reply by joranmulderij** [reply]
Oh I'm sorry if I came across as rude or someone who expects something from an open source developer. That was not actually my point. I meant that it is better that everything that is in the documentation should least work. I actually really like the documentation on this project. The documentation for dynamic queries is slightly off where you are using the unnamed constructor of `WhereClause` in your example; that was replaced by multiple subclasses.

Something that I personally like and I know that other people also value is the use of larger documentation comments. That way you don't have to go to the official documentation just to look up something like the 'silent' parameter. Having a small explanation as a comment is enough.

Please don't think that I'm expecting anything; thank you very much for contributing to the Flutter open source packages by writing by far the best database library for Flutter. Packages like these make Flutter more valuable for developers.

**Reply by simc** [reply]
I'll fix that 👍

**Reply by joranmulderij** [reply]
Found another one: sync get calls to the database don't seem to work on web. If this is indeed the case it should be in the list of web limitations.

**Reply by simc** [reply]
Yes, IndexedDB doesn't support synchronous access unfortunately. I'll list it as limitation
